Formula/z/zipkin.rb

```diff
@@ -0,0 +1,37 @@
+class Zipkin < Formula
+  desc "Collect and visualize traces written in Zipkin format"
+  homepage "https://zipkin.io"
+  url "https://search.maven.org/remotecontent?filepath=io/zipkin/zipkin-server/3.0.2/zipkin-server-3.0.2-exec.jar"
+  sha256 "390abe74dc4eb6d1ee27244da401361611b94f2c8ddb504dd4faa10095fbde77"
+  license "Apache-2.0"
+
+  livecheck do
+    url "https://search.maven.org/remotecontent?filepath=/io/zipkin/zipkin-server/maven-metadata.xml"
+    regex(%r{<version>v?(\d+(?:\.\d+)+)</version>}i)
```

Comment:
this would fix the livecheck issue
```suggestion
    url "https://search.maven.org/remotecontent?filepath=io/zipkin/zipkin-server/maven-metadata.xml"
    regex(%r{<version>v?(\d+(?:\.\d+)+)</version>}i)
```

Reply:
thanks. let me know if I should squash

Reply:
nm not squashing failed a rule, so I did it.